README.markdown

```diff
@@ -436,16 +436,27 @@ scripts.
 
 __4.12. Q. How can I check scripts written for different versions of Ruby?__
 
-A. Install a Ruby version manager such as [rvm][rvm] or [rbenv][rbenv],
-activate the environment for the relevant version of Ruby, and install in it
-the checkers you want to use.  Set `g:syntastic_ruby_checkers` accordingly in
-your `vimrc`, and run [Vim][vim] from the virtual environment.
-
-If you're starting Vim from a desktop manager rather than from a terminal you
-might need to write wrapper scripts around your checkers, to activate the
-virtual environment before running the actual checks.  Then you'll need to
-point the relevant `g:syntastic_ruby_<checker>_exec` variables to the wrapper
-scripts.
+A. Install a Ruby version manager such as [rvm][rvm],  [rbenv][rbenv],
+[chruby][chruby] or [ry][ry], activate the relevant version of Ruby, and
```

Comment:
I don't doubt `chruby` and `ry` do what they claim in a terminal.  What I'm asking is whether the effects of running them survive all the way to the linters run from Vim.  All methods you describe above incur limitations to syntastic and break some popular tricks related to changing `exe` or `exec`.

So, did you actually test them __with syntastic__, on at least Windows, Mac OS X (or some *BSD), and Linux?  Did you test them with different __shells__ (`ksh`, `bash`, `zsh`), and with different methods of __changing__ shells in Vim?